docs/src/pages/components/data-grid/components/components.md

```diff
@@ -7,10 +7,83 @@ components: DataGrid, XGrid
 
 <p class="description">The grid is highly customizable. Override components using the <code>components</code> prop.</p>
 
+## Installing Component
```

Comment:
Heading convention

```suggestion
## Installing component
```

Reply:
But the terminology might be confusing, not sure. No strong point of view, maybe this could work better:

```suggestion
## Overriding components
```

Reply:
+1 for "overriding". The first time I saw I thought it was installation instructions. 😁